src/core/mac/mac_frame.cpp

```diff
@@ -1048,6 +1048,16 @@ void Frame::CopyFrom(const Frame &aFromFrame)
     memcpy(mIeInfo, aFromFrame.mIeInfo, sizeof(otRadioIeInfo));
 }
 
+uint16_t Frame::GetMtu(void) const
```

Comment:
nit: since these are simple one-line method, suggest making these inline (adding them in header file)

Reply:
These will get extended with more advanced logic once mutli-radio support is contributed, so leaving the implementations here will simplify review and merge conflict resolution of the future feature diffs.